lib/fluent/plugin/out_lm.rb

```diff
@@ -71,6 +73,11 @@ def process_record(tag, time, record)
       resource_map = {}
       lm_event = {}
       lm_event["message"] = record["message"]
+      
+      if @force_encoding 
+        lm_event["message"] = lm_event["message"].force_encoding("ISO-8859-1").encode("UTF-8")
```

Comment:
Should `force_encoding` be a config option as well, in case you need to force encoding but know the charset to differ a lot from `iso-8859-1`?

Reply:
Great! I will make force encoding as string so you need to specify it.